spec/features/user_index_integration_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe 'User Index page', type: :system do
+  before(:each) do
+    driven_by(:rack_test)
+    first_user = User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                             bio: 'Teacher from Mexico.')
+    User.create(name: 'Lilly', photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+                bio: 'Teacher from Poland.')
+    Post.create(author: first_user, title: 'Hello', text: 'This is my first post')
+    visit root_path
+  end
+  describe 'checks the users' do
+    it 'shows the right names' do
+      expect(page).to have_content('Tom')
```

Comment:
- [ ] You are to test for all user's name, currently, you tested for only Tom and not for Lilly, kindly test for all user's name